__tests__/app.test.js

```diff
@@ -31,7 +31,53 @@ describe("GET /api/topics", () => {
       .get("/api/topics")
       .expect(200)
       .then(({ body }) => {
-        expect(body.topics).toEqual(data.topicData);
+        expect(body.topics).toHaveLength(3);
+        body.topics.forEach(({ description, slug }) => {
+          expect(typeof description).toBe("string");
+          expect(typeof slug).toBe("string")
+        })
       });
   });
-})
\ No newline at end of file
+})
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: Responds with an object of the article with the correct article id", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toHaveLength(1);
```

Comment:
As we know we only want the one article object, it makes sense to expect this back rather than expecting back an an array and then having to access that one object

Reply:
How would this change the way we assert in this test?